website/integrations/services/apache-guacamole/index.mdx

```diff
@@ -51,29 +59,49 @@ import TabItem from "@theme/TabItem";
     { label: 'Standalone', value: 'standalone' },
   ]}>
   <TabItem value="docker">
-    The Docker containers are configured via environment variables. The following variables are required:
-
-```yaml
-OPENID_AUTHORIZATION_ENDPOINT: https://authentik.company/application/o/authorize/
-OPENID_CLIENT_ID: # client ID from above
-OPENID_ISSUER: https://authentik.company/application/o/*Slug of the application from above*/
-OPENID_JWKS_ENDPOINT: https://authentik.company/application/o/*Slug of the application from above*/jwks/
-OPENID_REDIRECT_URI: https://guacamole.company/ # This must match the redirect URI above
-OPENID_USERNAME_CLAIM_TYPE: preferred_username
-```
+Docker containers are typically configured using environment variables. To ensure proper integration, add the following variables to your `.env` file:
+
+<IntegrationsMultilineCodeblock>
+    {`
+OPENID_AUTHORIZATION_ENDPOINT=https://<em>authentik.company</em>/application/o/authorize/
+OPENID_CLIENT_ID=<em>Client ID from authentik</em>
+OPENID_ISSUER=https://<em>authentik.company</em>/application/o/<em>your-slug</em>/
+OPENID_JWKS_ENDPOINT=https://<em>authentik.company</em>/application/o/<em>your-slug</em>/jwks/
+OPENID_REDIRECT_URI=https://<em>guacamole.company</em>/ # Must match Redirect URI in authentik
```

Comment:
Should this be italicized? Is that a variable, or exactly what they need to enter?